client/src/components/Dashboard.tsx

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import { TimeSeries, TimeEvent } from "pondjs";
+import {
+    Charts,
+    ChartContainer,
+    ChartRow,
+    YAxis,
+    LineChart
+} from "react-timeseries-charts";
+
+function Dashboard() {
+    const tempSeries = new TimeSeries({
+        name: "temp",
+        columns: ["time", "value"],
+        points: [
+            [1400425947000, 1000],
+            [1400425948000, 2000],
+        ],
+    });
+
+    const [altitudeData, setAltitudeData] = React.useState(tempSeries);
+    const [accelerationData, setAccelerationData] = React.useState(tempSeries);
+    const [engine1Data, setEngine1Data] = React.useState(tempSeries);
+    const [engine2Data, setEngine2Data] = React.useState(tempSeries);
+    const [engine3Data, setEngine3Data] = React.useState(tempSeries);
+    const [loading, setLoading] = React.useState(true);
+
+    React.useEffect(timeSeriesFormatAll);
+
+    // Format of data from endpoint 
+    interface Data {
+        result: string;
+        table: number;
+        _start: string;
+        _stop: string;
+        _time: number;
+        _value: number;
+        _field: string;
+        _measurement: string;
+        _flight: string;
+        _host: string;
+    }
+
+    interface HttpResponse<T> extends Response {
+        parsedBody?: T;
+    }
+
+    // Retrieves data; parses body 
+    async function http<T>(
+        request: RequestInfo
+    ): Promise<HttpResponse<T>> {
+        const response: HttpResponse<T> = await fetch(request);
```

Comment:
I'm not sure all of these need explicit types; most of them could probably be inferred!